distribution.gemspec

```diff
@@ -23,4 +23,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'bundler'
   s.add_development_dependency 'rake'
   s.add_development_dependency "rspec", '~> 3.2'
+  s.add_development_dependency 'rb-gsl'
```

Comment:
Can you remove this line as well? I'm trying to find the bug and need to merge this patch, but don't want to add rb-gsl as a dependency, as I said in another thread. :)